packages/aws-cdk/lib/api/cxapp/exec.ts

```diff
@@ -57,7 +62,7 @@ export async function execProgram(aws: SdkProvider, config: Configuration): Prom
     return createAssembly(app);
   }
 
-  const commandLine = await guessExecutable(appToArray(app));
+  const commandLine = await guessExecutable(commandToArray(app));
```

Comment:
Not sure this is better TBH...

What happens if the command has quotes in it?

For example `"mvn package"`?

Bottom line, I think we should simply spawn this command without splitting it into arguments and with `shell:true`. There's a variant of spawn I believe that just accepts a single string and passes it to the shell.

There are also intricacies related to Windows/POSIX here that can blow up in 5,000 ways, so I rather we avoid any parsing of the command line if possible.

Reply:
@eladb just to be clear - you also want to change how we handle the `"app"` key, right? And no longer do any splitting there?